py/torch_tensorrt/dynamo/_compiler.py

```diff
@@ -302,6 +301,24 @@ def compile_module(
             f"Detected support for {num_supported_ops} operators out of {total_ops} in subgraph."
         )
 
+    def contains_metadata(gm: torch.fx.GraphModule) -> bool:
+        for node in gm.graph.nodes:
+            if node.op != "output" and (not node.meta) and "val" not in node.meta:
```

Comment:
currently I'm relying on `"val"` key in the node.meta which has the fake tensors with symbolic shape/static shapes.
There's also `example_tensor` key sometimes but since I'm not using it, `and "val" not in node.meta` condition will make it strict.